crypto/sm2/sm2_sign.c

```diff
@@ -428,19 +428,19 @@ int sm2_sign(const unsigned char *dgst, int dgstlen,
 
     e = BN_bin2bn(dgst, dgstlen, NULL);
     if (e == NULL) {
-       SM2err(SM2_F_SM2_SIGN, ERR_R_BN_LIB);
+       SM2err(SM2_F_SM2_INTERNAL_SIGN, ERR_R_BN_LIB);
```

Comment:
We could just set these `F` codes to 0 rather than introducing new, redundant ones.